pkg/apply/applier_test.go

```diff
@@ -586,9 +585,12 @@ func TestApplier(t *testing.T) {
 						break loop
 					}
 					if e.Type == event.ActionGroupType &&
-						e.ActionGroupEvent.Action == event.ApplyAction &&
 						e.ActionGroupEvent.Type == event.Finished {
-						close(poller.start)
+						if e.ActionGroupEvent.Action == event.ApplyAction ||
```

Comment:
What are we doing here? It is not obvious.

Reply:
I'll add a comment. We need to consider `Prune` and `Delete` tasks now instead of just `Apply` tasks.